src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingActualsResponseMapper.java

```diff
@@ -43,33 +43,6 @@ public HearingActualResponse toHearingActualResponse(HearingEntity hearingEntity
         return response;
     }
 
-    private String getHearingStatus(HearingEntity hearingEntity) {
```

Comment:
As this app is still work in progress, better remove `private` then the method itself, if the method is unused.